index.html

```diff
@@ -2810,6 +2813,84 @@ <h2 class="icon-title">
           </div>
         </section>
       </section>
+      <section id="solid-color-icons">
+        <h2>
+          Solid color icons
+        </h2>
+        <p>
+          Some platforms enforce that icons be displayed with a single color
+          or gradient, where only the transparency of the icon can be
+          controlled. As web applications should work across multiple
+          platforms, it it possible to indicate that an icon can have a
+          user-agent-specified color applied by adding the <a>monochrome</a>
+          purpose. This allows the platform to ensure that the icon looks well
+          integrated with the platform, and even apply different colors and
+          padding in different places throughout the platform.
+        </p>
+        <p>
+          When processing a <a>monochrome</a> icon, the user agent MUST NOT use
```

Comment:
Doesn't the OS do the color conversion defined here? Or do we need an algorithm to convert images before we hand them to the OS... that's gonna be tricky.

Reply:
I'll need to investigate what's done here. On Android, it's done automatically for notifications but not shortcuts.

Reply:
Windows converts monochrome icons automatically in the jump list.